client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -244,6 +248,24 @@ public <K, V, C> ShuffleHandle registerShuffle(int shuffleId, ShuffleDependency<
       id.compareAndSet(null, SparkEnv.get().conf().getAppId() + System.currentTimeMillis());
     }
     LOG.info("Generate application id used in rss: " + id.get());
+
+    String storageType = sparkConf.get(RssClientConfig.RSS_STORAGE_TYPE);
```

Comment:
A question
If user configure this storagePath with `/userA/xxxxx` instead of `hdfs://xxxx:999/userA/xxxxx`, maybe client and shuffle server have different default hdfs Filesystem.

Reply:
The remoteStorage should be configured by coordinator. If user do the configuration, it will be overrided.
So, make sure configuration in coordinator is formatted as `hdfs://xxxx:999/userA/xxxxx`, and there will be no misunderstanding for shuffle server

Reply:
ok